recipes/_source_nrpe.rb

```diff
@@ -77,3 +77,16 @@
   EOH
   not_if "nrpe --version | grep #{node['nrpe']['version']}"
 end
+
+directory node['nrpe']['pid_dir'] do
```

Comment:
You can use the existing attribute and File.dirname(THAT_ATTRIBUTE) to get the name of the folder containing the pid without the need for another attribute